specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabricManagedClusters/preview/2021-01-01-preview/examples/ManagedClusterPatchOperation_example.json

```diff
@@ -0,0 +1,101 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "resRg",
+    "clusterName": "myCluster",
+    "api-version": "2021-01-01-preview",
+    "parameters": {
+      "tags": {
```

Comment:
@pilor we currently only support Patch requests to update tags as its required by arm https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#patch-resource. Is it okay to only have the tags in the request, or does it need anything else here like the location?

Reply:
just tags is fine